### PR TITLE
feat(phone-input): replace flag svg with emoji

### PR DESCRIPTION
**Pull request by nadiia**
#### Description

Replaces default flag svgs usage with emojis. It results in the phone-input size reduction from 136.17 KB to 10.79 KB.

#### Scope

- [ ] Patch: Bug Fix
- [x] Minor: New Feature
- [ ] Major: Breaking Change

### CODE REVIEW

File: src/phone-input/styled-components.js [hunk 1]
```diff
@@ -15,22 +13,23 @@ import {
...
 import defaultProps from '../select/default-props.js';
+import type {SizeT} from './types.js';
 
 type SizeStyleProps = {
-  $size?: $Keys<typeof SIZE>,
+  $size?: SizeT,
 };
 type HeightStyleProps = {$height: string};
 
-export const StyledFlag = styled<typeof Flag, SizeStyleProps>(
```
Comment (chasestarr) [inline src/phone-input/styled-components.js]: `StyledFlag` export needs to be maintained

Comment (chasestarr) [reply src/phone-input/styled-components.js]: otherwise, lgtm

**Comment by houmark** [comment]
This is great, but it's gonna conflict with #1717 which is near done also! I hope it doesn't slow that one down and I hope you core guys will put an effort into reviewing and helping to get that one merged as it will really improve the usability of this component.

File: src/phone-input/styled-components.js (hunk 1 above)
Comment (nadiia) [reply src/phone-input/styled-components.js]: I moved it to the `flag.js` and still export from `index`

**Comment by mihaisavezi** [comment]
Just reopening this old issue since this is not supported on windows devices. Would be good to have a headsup in the documentation, or revert to old version.

"Microsoft doesn't support any country flags on Windows, instead showing the two-letter country codes". - https://newbedev.com/flag-emojis-not-rendering

**Pull request merged**